providers/persister/inventory_collections.md

```diff
@@ -0,0 +1,292 @@
+# Inventory Collection
+2018/06/19
+
+[back](../dev-guide.md)
+
+**Inventory Collection** (```ManagerRefresh::InventoryCollection``` - next called **"IC"**) is an object used by Refresh process. 
+It represents properties of inventory like **VM**, **Hardware**, **Operating system**, **Host** etc.
```

Comment:
> It represents properties of inventory like

How about `It represents types of inventory such as ...`?  When I think of properties of inventory I think of e.g. cpus/memory not whole classes of objects

Reply:
I mean IC defines properties of inventory. As an inventory I mean VM, HW etc.
--> my lng skills :)

Reply:
Haha no worries :)